app/Services/PackageLoaderService.php

```diff
@@ -9,13 +9,30 @@ public static function menuPackages()
         $packagesList = array_pluck(config('packages'), 'name');
         foreach ($packagesList as $package) {
             $items[] = [
-                'url' => route(config('packages.' . $package . '.url')),
-                'caption' => config('packages.' . $package . '.caption'),
-                'icon' => config('packages.' . $package . '.icon'),
-                'active' => config('packages.' . $package . '.active'),
+                'url' => route(config('packages.' . $package . '.menu.url')),
```

Comment:
please remove spaces around concatenation
`'packages.'.$package.'.menu.url'`
This is PSR rule